### ISSUE TITLE
reported loss during finetuning

### ISSUE THREAD

**Issue by mmmddd98** [vdemichev/DiaNN#2015]
Hi DIA-NN team,

I am currently looking into fine-tuning in DIA-NN and noticed that the log reports loss values such as:

`2%: loss = 0.0578682`

Could you clarify whether this value represents the training loss or the validation loss?

I would also be interested to know how the loss is calculated. Is it, for example, a MSE between the predicted and observed values, or is a different loss function used?

Thank you very much for your help.

**Comment by vdemichev** [comment]
Hi,

I think it's training loss per batch. So it's expected to be noisy. For RT or IM it's transformed MSE, for fragmentation it's a quite complex function.

Best,
Vadim

**Comment by mmmddd98** [comment]
Thank you very much for the quick and helpful response!
